.github/workflows/ci.yml

```diff
@@ -108,3 +108,55 @@ jobs:
         with:
           name: docs
           path: doc/build/html
+
+  build_wheels:
+    name: Build Python wheel packages on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Fetch latest BuildBox release
+        run: ${GITHUB_WORKSPACE}/.github/wheel-helpers/fetch-latest-buildbox-release.sh
+
+      - name: Build wheels
+        run: pipx run cibuildwheel==2.8.1
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: wheels
+          path: ./wheelhouse/*.whl
+
+  test_wheels:
+    name: Test Python wheel packages on ${{ matrix.os }}
```

Comment:
This is not working correctly, I wonder if `Test Python wheel packages on "${{ matrix.os }}"` might work.

Currently jobs displayed on this PR display 4 different instances of just `Test Python wheel packages on` which looks confusing.

Minor issue, but would be useful to see it in the UI nonetheless.

Reply:
The issue that the `os` variable is not used at all,  but a different variable named `test-name` :) fixed now